java/core/src/java/org/apache/orc/OrcFile.java

```diff
@@ -905,6 +907,10 @@ public int getRowIndexStride() {
       return rowIndexStrideValue;
     }
 
+    public boolean isBuildIndex() {
```

Comment:
Yeah. we could re-implement it by rowIndexStrideValue> 0

Reply:
Here is the PR with your authorship.
- https://github.com/apache/orc/pull/1371

Reply:
> @deshanxiao It seems that we cannot revert this because this is a `public` API.

Agree, it looks like we can only set `orc.row.index.stride` to 0 when `orc.create.index=false`

Reply:
Thank you, @guiyanakuang .